### PR TITLE
news218: add French translation

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

**Comment by Copinmalin** [comment]
Hi @jonatack , A new translation.

I have some doubt, give me your point of view about : 
  
a trusted setup --> installation en confiance ?
Bitcoin's contracting language --> language de programmation de contrat de Bitcoin

And I imagine some other topic

Many thanks for your support

**Comment by jonatack** [comment]
> a trusted setup --> installation en confiance ? Bitcoin's contracting language --> language de programmation de contrat de Bitcoin

Try twitter polls for these as well (along with blocks versus blocs).

File: _posts/fr/newsletters/2022-09-21-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+title: 'Bitcoin Optech Newsletter #218'
+permalink: /fr/newsletters/2022/09/21/
+name: 2022-09-21-newsletter-fr
+slug: 2022-09-21-newsletter-fr
+type: newsletter-fr
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2022-09-21-newsletter.md:6]:
```suggestion
type: newsletter
```

File: _posts/fr/newsletters/2022-09-21-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+layout: newsletter-fr
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2022-09-21-newsletter.md:7]:
```suggestion
layout: newsletter
```

File: _posts/fr/newsletters/2022-09-21-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+lang: fr
+---
+Cette semaine la newsletter résume une discussion à propos de l'utilisation
+de `SIGHASH_ANYPREVOUT` pour reproduire des aspects des chaines dérivées.
+Sont inclus aussi nos sections habituelles abordant les derniers changements
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2022-09-21-newsletter.md:12]: Suggest to remove trailing space on this line

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
+---
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:1]: Can remove this file from this PR since we merged fr-217 previously

**Review by bitschmidty — CHANGES_REQUESTED** [review]
Thanks for the translation draft, @Copinmalin !

File: _posts/fr/newsletters/2022-09-21-newsletter.md (hunk 3 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-21-newsletter.md:12]: Done

File: _posts/fr/newsletters/2022-09-14-newsletter.md (hunk 4 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-14-newsletter.md:1]: I don't understand your demand, please can you explain what you mean ?

The file "2022-09-14-newsletter.md" should stay in the folder "newsletters" for sync, no ?

**Comment by bitschmidty** [comment]
Apologies, I have rebased this PR on master and resolved the extraneous commits that didn’t apply to the news218 French translation.

**Comment by bitschmidty** [comment]
Added a link fix

**Review by bitschmidty — APPROVED** [review]
@Copinmalin I have some suggestions to improve the workflow for French translations:

- pull the latest from bitcoinops:master branch
- create a separate branch ("fr-translate-218" to use the azuchi's Japanese branch name format for example) for each PR off of bitcoinops:master
- run `make production` locally to catch any build/lint errors
- Create the PR from your branch (and not master)

What do you think?

**Comment by Copinmalin** [comment]
Hello @bitschmidty ,

I create PR in a new branch in my repository as needed :

https://github.com/Copinmalin/bitcoinops.github.io/pull/3

How to for reviewing by you ?

**Pull request merged**